Automattic/wp-calypso
# Create a standalone i18n-calypso-cli package

This PR removes the CLI tool from `i18n-calypso` package and moves it to a new `i18n-calypso-cli`. Technically, they have very little in common: one is a runtime dependency, the other one is a build-time tool. Their NPM dependencies are very different. One has a build step and ships transpiled files, the other one is a Node module and ships as-is.

I also rearranged the `i18n-calypso-cli` files so that the translation extraction tool can be used as a library, too (in addition to the CLI command):
```js
const i18nCLI = require( 'i18n-calypso-cli' );
i18nCLI( { ...options } );
```

That should fix issues found in Automattic/jetpack#13705

**How to test:**
Verify that things like `npm run translate` continue to work. And that that Jetpack issue is really fixed by this.
